spell/spell-reviewer-goerli-checklist.md

```diff
@@ -121,10 +119,11 @@ Spell Actions:
     * [ ] Cliff Duration matches Doc (`eta`)
     * [ ] Restricted (by default)
     * [ ] Manager match Doc (`mgr`, set to zero for DAI streams by default)
+    * [ ] IF `DssVestTransferrable` is used
+      * [ ] Ensure `DssVestTransferrable` Allowance matches total transfer amount
```

Comment:
I kept getting stuck on this, can you please elaborate how I would go about checking this? Is there a contract you're reading a value off?

Reply:
yeah, this could be rephrased better

Reply:
done in 9e2dcf5 (see for references [spell archive](https://github.com/makerdao/spells-mainnet/blob/master/archive/2022-10-05-DssSpell/DssSpell.sol#L88) and [vest contract](https://github.com/makerdao/dss-vest/blob/master/src/DssVest.sol#L469)).